### PR TITLE
SC-25 # Updated serverless wrapper to use lambda proxy syntax

### PR DESCRIPTION
**Pull request by mymattcarroll**
### Added
- SC-25: `--port` flag to `bm server serve` cmd
### Changed
- SC-25: Serverless wrapper handler to accommodate for `lambda-proxy` integration
### Code Review Notes
- Known incomplete work to be tackled in future:
  - [ ] Allow for customer to set there own status code and headers (including CORS)
  - [ ] Updated error handling to output more information when errors thrown.

### CODE REVIEW

File: bin/index.js [hunk 1]
```diff
@@ -49,7 +51,7 @@ try {
   main = require(path.join(__dirname, '..', 'commands', `${command}.js`))
 } catch (err) {
   console.error(chalk.red(`
-Unknown command: ${command}`))
+Unknown command: ${command}`, err))
```
Comment (jokeyrhyme) [inline bin/index.js]: We should aim to never expose customers to stack traces and other raw errors.

Perhaps we should follow a pattern seen elsewhere and have a `--debug` flag that enables this sort of thing?

Comment (mymattcarroll) [reply bin/index.js]: Ahh crap, didnt mean to check this in.

I do like the new flag idea, would you prefer `--debug` over `--verbose`

Comment (jokeyrhyme) [reply bin/index.js]: I considered `--verbose`. I think I prefer `--debug` for this based on my definitions:
- verbose: more information
- debug: information you need when things are broken

Comment (mymattcarroll) [reply bin/index.js]: Yeah agreed, my only worry is the buildbot cli currently already has a `--debug` flag for the `bm buildbot build` cmd. So prob wont be able to make this a consistent pattern without changing the buildbot cmd

Comment (jokeyrhyme) [reply bin/index.js]: Yeah, does someone wanting `cordova build --debug` also want to see stack traces and other internal data from the CLI tool? Probably not. Alternatively, we always do both debug and production builds, giving users both APKs. Then it frees up the flag for other purposes. /shrug

Comment (mymattcarroll) [reply bin/index.js]: Might just leave off for now.

**Comment by jokeyrhyme** [comment]
LGTM 👍

**Comment by mymattcarroll** [comment]
### Added

- SC-25: `--port` flag to `bm server serve` cmd

### Changed 

- SC-25: Serverless wrapper handler to accommodate for `lambda-proxy` integration

**Pull request merged**